.github/workflows/rabbit_consumer_schedule.yaml

```diff
@@ -0,0 +1,120 @@
+name: Rabbit Consumer Schedule
+
+# Incrementing the patch version on a schedule to pull latest containers dependencies.
+
+
+on:
+  schedule: 
+      - cron: "12 10 * * TUE" # Every Tuesday at 10.12am
```

Comment:
why 10.12am? :)

Reply:
Also I'm not sure we need a schedule? 
Why not make it so this workflow runs when a PR is merged into master instead? 
Like a dependabot PR

Reply:
If changing it to only run when PR is merged - It should only increment the version if the file hasn't already been changed manually - something like this might happen if we make a major change and need to update the minor/major version

Reply:
10.12am was sort of random but I chose Tuesday in the morning so that if there are any problems we can catch them and they are not left for after the weekend or at the end of the week.

I think we chose to schedule it just to keep on top of the dependency updates without having to check ourselves. 

We also do still have the original rabbit_consumer.yaml file that runs the workflow when we merge a PR into master, this is a duplicate file solely for scheduling. If we do do a minor or major change, we'll have to update version accordingly ourselves. Hopefully that makes sense :)